[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\minhn\\AppData\\Local\\Temp\\quarto-session5011ef0c48d2a051'\n"]
